authority-portal-backend/authority-portal-api/src/main/java/de/sovity/authorityportal/api/model/ConnectorOverviewEntryDto.kt

```diff
@@ -19,6 +19,8 @@ import io.swagger.v3.oas.annotations.media.Schema
 data class ConnectorOverviewEntryDto(
     @field:Schema(description = "Connector ID", requiredMode = Schema.RequiredMode.REQUIRED)
     val id: String,
+    @field:Schema(description = "Organization Name", requiredMode = Schema.RequiredMode.REQUIRED)
+    val customerOrgName: String,
```

Comment:
Since this DTO is reused also for self-hosted connectors, "customer" might be a little misleading.
How about something like `owningOrgName` or just `organizationName`?